Resources/public/javascript/forms.js

```diff
@@ -1,16 +1,61 @@
+let removePopup = `<div class="sonata-ba-delete popup-file-delete mfp-hide">
```

Comment:
This goes for some other stuff as well, for instance the text counter, when characters are limited. So how do we handle translations now?